### PR TITLE
Create Stack.java

### PR DESCRIPTION
**Pull request by Udit-singh**

### CODE REVIEW

File: Stack.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
+package stack;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/*
```
Comment (kaustubhgupta) [inline Stack.java:5]: Put these comments on the top

File: Stack.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+		public Type2 val;
+		public SListNode<Type2> next;
+		
+		public SListNode(Type2 i) {
+			this.val = i;
+			this.next = null;
+		}
+	}
+		
+	public Stack() {
```
Comment (kaustubhgupta) [inline Stack.java:23]: Explain your functions with appropriate comments

File: Stack.java [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+		return str;
+	}
+    
+	public static void main(String[] args) {
+		
+		Stack<Character> stack = new Stack<>();
+
+		System.out.println("stack size: " + stack.size() + "\n");
+		
+		stack.push('i');
```
Comment (kaustubhgupta) [inline Stack.java:98]: Instead of hardcoding, take inputs from user

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: Stack.java (hunk 2 above)
Comment (YashSaxena9) [reply Stack.java:23]: It is necessary to add docStrings(documentation like multiline comments) at the top of your functions and methods. Please add them as per need

**Review by YashSaxena9 — COMMENTED** [review]
The following code matches our criteria, we will be happy to merge your PR to our main branch. Please refactor the changes suggested so that we can merge the PR.

**Comment by kaustubhgupta** [comment]
@Udit-singh due to your inactivity this PR has been closed. If you still wish to contribute then read the guidelines and make new PR

**Pull request closed without merging**